### PR TITLE
PR Review 

### PR DESCRIPTION
**Pull request by udberg**
We(@KCnick and @udberg ) Implemented a clone of the New York Times article. We utilized knowledge acquired from HTML5 and CSS Blocks and Flexbox to achieve the desired UI. We opted to close the previously open PR because we opted for a completely different approach than the one we used a couple hours ago.

### CODE REVIEW

File: .gitignore
Comment (descholar-ceo) [inline .gitignore]: Remove this package-lock.json file from gitignore

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @udberg and @KCnick  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes]

### :red_circle:  Next time please, don't merge or close the current PR before it get approved, don't do that anymore :red_circle:
- [ ] This branch has conflicts with the base branch, kindly resolve them, you can use [this medium article](https://medium.com/@emmamugira/how-to-resolve-git-conflicts-after-rebase-e4afde1eab1) to know how to fix conflicts from github
- [ ] :point_right: It looks like you have difficult to understand what a live demo link means, here is a simple expanation about it: It is a link which redirect to the live version of your project, in that case you need to deploy it using some free services such as [Githack](https://raw.githack.com/), Github pages, [netlify](https://www.netlify.com/), or [heroku](https://dashboard.heroku.com/). After deploying it, you will get a link which represent the online live version of your project, that is the one you need to copy and paste on `Deployment` or `Live Demo`, the current link you have put is broken, please learn more about the deployment
- [ ] The license link from your readme file doesn't work, fix that before re-submitting
- [ ] The link to gtihub profile of @KCnicks is not working 
- [ ] Make the Pull Request's name a meanigful One, currently its name is PR Review, that name doesn't reflect the actual work you have done on this Pull Request
- [ ] The same thing applies to the branch name, currently your branch name is `udberg-patch-1`, this branch name doesn't follow standards of descriptive name. Please change it into something meaningful
- [ ] The getting started section in your readme file, has some errors.
:warning: It's not `cd new-york times article` it should be `cd clone-new-york-times-article`
:warning: It's not `git checkout -b article-clone` it should be `git checkout udberg-patch-1`
:x: The `git pull` is useless here, after cloning you have all latest changes on the branch, you only need to check it out :point_up: previous step
:warning: Most of the time `live-server` runs on `port 5500` so this step `If you have vscode live-server installed, run it and open localhost:3000` shouldn't be specifying the port to open, it might not work for some reasons, it's better to not specify the port number
:x:  The `setup` section is useless, since you already elaborated the steps to follow to get a local copy up and running
:warning: The `Run NPM locally Run tests to check the HTML and CSS files.` command doesn't work, you should tell a user that he/she needs to install needed dependencies by running this command `npm install --save-dev stylelint@13.3.x stylelint-scss@3.17.x stylelint-config-standard@20.0.x stylelint-csstree-validator` and then after they need to run `npx stylelint "**/*.{css,scss}"` command to check linters if they need.
- [ ] Remove the `package-lock.json` from the `.gitingore` file

=======
If anything seems to be unclear to you, comment on this Pull Request and mention me as [@descholar-ceo](https://github.com/descholar-ceo)
[LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

File: .gitignore
Comment (udberg) [reply .gitignore]: @descholar-ceo thank you so much. This is the most helpful review we've had all day with resources enough to help us understand how best to progress. It should be fixed soon. Again, thank you.

**Comment by mimipeshy** [comment]
## STATUS : CHANGES REQUIRED :red_circle:  :recycle: 

Hello team :wave:  :wave: , you guys have done excpetionally well in implementing the changes highlted above :clap:  :clap: , 
I really wanted to approve this project but there is something you forgot to check 

- [ ]  The setup section is useless since you already elaborated the steps to follow to get a local copy up and running
warning The Run NPM locally Run tests to check the HTML and CSS files. the command doesn't work, you should tell a user that he/she needs to install needed dependencies by running this command npm install --save-dev stylelint@13.3.x stylelint-scss@3.17.x stylelint-config-standard@20.0.x stylelint-csstree-validator and then after they need to run npx stylelint "**/*.{css,scss}" command to check linters if they need.
- Please remove the Linters  section

That's all :relaxed:  , please submit another review when done.

Regards 
Peris Ndanu
Feel free to slack me if you have any more questions regarding this.

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on implementing all project requirements and getting this project approved. :+1:

You can merge this branch into the main branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime**

**Pull request merged**
